Engines/Wine/Verbs/msxml6/script.js

```diff
@@ -0,0 +1,57 @@
+include("engines.wine.engine.object");
+include("engines.wine.plugins.override_dll");
+include("utils.functions.net.resource");
+include("utils.functions.filesystem.files");
+
+
+/**
+* Verb to install msxml6
+* @returns {Wine} Wine object
+*/
+Wine.prototype.msxml6 = function () {
+    if (this.architecture() == "amd64") {
+        var setupFile64 = new Resource()
+            .wizard(this.wizard())
+            .url("https://download.microsoft.com/download/e/a/f/eafb8ee7-667d-4e30-bb39-4694b5b3006f/msxml6_x64.msi")
+            .checksum("ca0c0814a9c7024583edb997296aad7cb0a3cbf7")
+            .name("msxml6_x64.msi")
+            .get();
+    }
+    else {
+        var setupFile32 = new Resource()
+            .wizard(this.wizard())
+            .url("https://download.microsoft.com/download/e/a/f/eafb8ee7-667d-4e30-bb39-4694b5b3006f/msxml6_x86.msi")
+            .checksum("5125220e985b33c946bbf9f60e2b222c7570bfa2")
+            .name("msxml6_x86.msi")
+            .get();
+    }
+    remove(this.system32directory() + "/msxml6.dll")
+    this.overrideDLL()
+        .set("native,builtin", ["msxml6"])
+        .do();
+    if (this.architecture() == "amd64") {
+    this.wizard().wait(tr("Please wait while {0} is installed...", "msxml6"));
```

Comment:
Winetricks also removes the 64 bits dlls in that case. Also, the indentation looks wrong.